lib/libzpool/util.c

```diff
@@ -159,7 +159,7 @@ set_global_var(char *arg)
 	char *varname = arg, *varval;
 	u_longlong_t val;
 
-#ifndef _LITTLE_ENDIAN
+#ifndef _ZFS_LITTLE_ENDIAN
```

Comment:
It looks like `zdb_dump_block`  in `zdb.c` needs to be updated as well.  The `_ZFS` prefix was added by commit 5678d3f5 to resolve a conflict, it looks like we missed a couple instances.